packages/data/json/terms.json

```diff
@@ -92,75 +96,101 @@
+      },
+      {
+        "definition": "E - <defined-term-glossary override-term='mercantile occupancy'>Mercantile occupancies</defined-term-glossary>"
+      },
+      {
+        "definition": "F1 - <defined-term-glossary override-term='high-hazard industrial occupancy'>High-hazard industrial occupancies</defined-term-glossary>"
+      },
+      {
+        "definition": "F2 - <defined-term-glossary override-term='medium-hazard industrial occupancy'>Medium-hazard industrial occupancies</defined-term-glossary>"
+      },
+      {
+        "definition": "F3 - <defined-term-glossary override-term='low-hazard industrial occupancy'>Low-hazard industrial occupancies</defined-term-glossary>"
+      }
     ]
   },
   "means of egress": {
     "definition": "<glossary-term>Means of egress</glossary-term> means a continuous path of travel provided for the escape of persons from any point in a <defined-term-glossary>building</defined-term-glossary> or contained open space to a separate <defined-term-glossary>building</defined-term-glossary>, an open public thoroughfare, or an exterior open space protected from fire exposure from the <defined-term-glossary>building</defined-term-glossary> and having access to an open public thoroughfare. Means of egress includes <defined-term-glossary override-term='exit'>exits</defined-term-glossary> and <defined-term-glossary override-term='access to exit'>access to exits</defined-term-glossary>.",
     "cleanDefinition": "Means of egress means a continuous path of travel provided for the escape of persons from any point in a building or contained open space to a separate building, an open public thoroughfare, or an exterior open space protected from fire exposure from the building and having access to an open public thoroughfare. Means of egress includes exits and access to exits."
   },
   "medium-hazard industrial occupancy": {
-    "definition": "<glossary-term>Medium-hazard Industrial Occupancies</glossary-term> <defined-term-glossary override-term='major occupancy' override-tooltip='group f division 2'>(Group F, Division 2)</defined-term-glossary> means an industrial <defined-term-glossary>occupancy</defined-term-glossary> in which the combustible content is more than 50 kg/m² or 1,200 MJ/m² of <defined-term-glossary>floor area</defined-term-glossary> and not classified as a high-hazard industrial occupancy.",
+    "definition": "<glossary-term>Medium-hazard industrial occupancies</glossary-term> <defined-term-glossary override-term='major occupancy' override-tooltip='group f division 2'>(Group F, Division 2)</defined-term-glossary> means an industrial <defined-term-glossary>occupancy</defined-term-glossary> in which the combustible content is more than 50 kg/m² or 1,200 MJ/m² of <defined-term-glossary>floor area</defined-term-glossary> and not classified as a high-hazard industrial occupancy.",
```

Comment:
This also needs the same update as `low-hazard` where the defined term is `industrial occupancy` instead of just `occupancy`.